quodlibet/quodlibet/_main.py

```diff
@@ -19,7 +19,7 @@
 
 
 PLUGIN_DIRS = ["editing", "events", "playorder", "songsmenu", "playlist",
-               "gstreamer", "covers", "query"]
+               "gstreamer", "covers", "query", "ui"]
```

Comment:
So far I guess nothing can be _just_ a UI plugin - that's more of a mixin than a full type - they nearly always end up being `EventPlugin`s too...

Reply:
i'll ditch 'ui' then, although it was more a chosen folder name for the 'PluginsWidgetBarPlugin', which to me 'works', but then as say it is just 'Advanced Preferences', and it's an eventplugin so ../events